test/widgets/subscription_list_test.dart

```diff
@@ -57,6 +59,23 @@ void main() {
     check(isUnpinnedHeaderInTree()).isFalse();
   });
 
+  testWidgets('link to other channels is shown', (tester) async {
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
+    final streams = List.generate(5, (index) => eg.stream());
+    await setupStreamListPage(tester,
+      streams: streams,
+      subscriptions: [eg.subscription(streams[1])]);
+
+    check(find.text(zulipLocalizations.browseMoreNChannels(4)).evaluate()).isNotEmpty();
```

Comment:
We can use the actual localized strings we expect here, because it will also be a bug if the pluralization does not work as expected.

Reply:
Yeah — when there's nontrivial logic around what the string should look like, that's a good case for having the test (or at least one test) write the actual content it expects from the string, rather than calling the same ZulipLocalizations method that the code under test will be calling.